### PR TITLE
Update Jenkinsfile - Closes #6308 #6310

### LINKED ISSUE #6308: Unstable functional tests for lisk-forging-plugin
### Expected behavior

The functional tests for Lisk Forging Plugin must not fail randomly. 

### Actual behavior

Sometimes these fail with following error: 

```
@liskhq/lisk-framework-forger-plugin: PASS test/functional/forger_info/event_track.spec.ts (57.001 s)
@liskhq/lisk-framework-forger-plugin:   ● Console
@liskhq/lisk-framework-forger-plugin:     console.error
@liskhq/lisk-framework-forger-plugin:       Unhandled error
@liskhq/lisk-framework-forger-plugin:       at process.uncaught (../../node_modules/jest-jasmine2/build/jasmine/Env.js:248:21)
@liskhq/lisk-framework-forger-plugin:     console.error
@liskhq/lisk-framework-forger-plugin:       Error: listen EADDRINUSE: address already in use 0.0.0.0:5000
@liskhq/lisk-framework-forger-plugin:           at Server.setupListenHandle [as _listen2] (net.js:1316:16)
@liskhq/lisk-framework-forger-plugin:           at listenInCluster (net.js:1364:12)
@liskhq/lisk-framework-forger-plugin:           at doListen (net.js:1501:7)
@liskhq/lisk-framework-forger-plugin:           at processTicksAndRejections (internal/process/task_queues.js:85:21)
@liskhq/lisk-framework-forger-plugin:       at process.uncaught (../../node_modules/jest-jasmine2/build/jasmine/Env.js:249:21)
@liskhq/lisk-framework-forger-plugin: FAIL test/functional/forger_info/forger_info_sync.spec.ts (6.99 s)
@liskhq/lisk-framework-forger-plugin:   ● Console
@liskhq/lisk-framework-forger-plugin:     console.error
@liskhq/lisk-framework-forger-plugin:       Unhandled error
@liskhq/lisk-framework-forger-plugin:       at process.uncaught (../../node_modules/jest-jasmine2/build/jasmine/Env.js:248:21)
@liskhq/lisk-framework-forger-plugin:     console.error
@liskhq/lisk-framework-forger-plugin:       Error: listen EADDRINUSE: address already in use 0.0.0.0:5000
@liskhq/lisk-framework-forger-plugin:           at Server.setupListenHandle [as _listen2] (net.js:1316:16)
@liskhq/lisk-framework-forger-plugin:           at listenInCluster (net.js:1364:12)
@liskhq/lisk-framework-forger-plugin:           at doListen (net.js:1501:7)
@liskhq/lisk-framework-forger-plugin:           at processTicksAndRejections (internal/process/task_queues.js:85:21)
@liskhq/lisk-framework-forger-plugin:       at process.uncaught (../../node_modules/jest-jasmine2/build/jasmine/Env.js:249:21)
@liskhq/lisk-framework-forger-plugin:     console.error
@liskhq/lisk-framework-forger-plugin:       Unhandled error
@liskhq/lisk-framework-forger-plugin:       at process.uncaught (../../node_modules/jest-jasmine2/build/jasmine/Env.js:248:21)
@liskhq/lisk-framework-forger-plugin:     console.error
@liskhq/lisk-framework-forger-plugin:       ReadError: Database is not open
@liskhq/lisk-framework-forger-plugin:           at maybeError (/home/lisk/workspace/lisk-sdk_PR-6294/node_modules/levelup/lib/levelup.js:339:32)
@liskhq/lisk-framework-forger-plugin:           at LevelUP.Object.<anonymous>.LevelUP.put (/home/lisk/workspace/lisk-sdk_PR-6294/node_modules/levelup/lib/levelup.js:211:7)
@liskhq/lisk-framework-forger-plugin:           at KVStore.put (/home/lisk/workspace/lisk-sdk_PR-6294/node_modules/@liskhq/lisk-db/src/kv_store.ts:98:18)
@liskhq/lisk-framework-forger-plugin:           at Object.setForgerSyncInfo (/home/lisk/workspace/lisk-sdk_PR-6294/framework-plugins/lisk-framework-forger-plugin/src/db.ts:51:11)
@liskhq/lisk-framework-forger-plugin:           at ForgerPlugin._syncForgerInfo (/home/lisk/workspace/lisk-sdk_PR-6294/framework-plugins/lisk-framework-forger-plugin/src/forger_plugin.ts:229:9)
@liskhq/lisk-framework-forger-plugin:           at processTicksAndRejections (internal/process/task_queues.js:97:5)
@liskhq/lisk-framework-forger-plugin:           at Immediate.<anonymous> (/home/lisk/workspace/lisk-sdk_PR-6294/framework-plugins/lisk-framework-forger-plugin/src/forger_plugin.ts:140:4)
@liskhq/lisk-framework-forger-plugin:       at process.uncaught (../../node_modules/jest-jasmine2/build/jasmine/Env.js:249:21)
@liskhq/lisk-framework-forger-plugin:   ● Forger Info Sync › should sync information from scratch on startup
@liskhq/lisk-framework-forger-plugin:     listen EADDRINUSE: address already in use 0.0.0.0:5000
```

### Steps to reproduce

Run tests suit on Jenkins

### Which version(s) does this affect? (Environment, OS, etc...)

development

### LINKED ISSUE #6310: Unstable functional tests for lisk-http-api-plugin
### Expected behavior

The functional tests for Lisk HTTP API Plugin must not fail randomly. 

### Actual behavior

Sometimes these fail with following error: 

```
@liskhq/lisk-framework-http-api-plugin: PASS test/functional/transactions.spec.ts (27.289 s)
@liskhq/lisk-framework-http-api-plugin:   ● Console
@liskhq/lisk-framework-http-api-plugin:     console.error
@liskhq/lisk-framework-http-api-plugin:       Unhandled error
@liskhq/lisk-framework-http-api-plugin:       at process.uncaught (../../node_modules/jest-jasmine2/build/jasmine/Env.js:248:21)
@liskhq/lisk-framework-http-api-plugin:     console.error
@liskhq/lisk-framework-http-api-plugin:       Error: listen EADDRINUSE: address already in use 0.0.0.0:5000
@liskhq/lisk-framework-http-api-plugin:           at Server.setupListenHandle [as _listen2] (net.js:1316:16)
@liskhq/lisk-framework-http-api-plugin:           at listenInCluster (net.js:1364:12)
@liskhq/lisk-framework-http-api-plugin:           at doListen (net.js:1501:7)
@liskhq/lisk-framework-http-api-plugin:           at processTicksAndRejections (internal/process/task_queues.js:85:21)
@liskhq/lisk-framework-http-api-plugin:       at process.uncaught (../../node_modules/jest-jasmine2/build/jasmine/Env.js:249:21)
@liskhq/lisk-framework-http-api-plugin: PASS test/functional/forging.spec.ts (17.349 s)
@liskhq/lisk-framework-forger-plugin: PASS test/functional/forger_info/event_track.spec.ts (57.271 s)
@liskhq/lisk-framework-forger-plugin:   ● Console
@liskhq/lisk-framework-forger-plugin:     console.error
@liskhq/lisk-framework-forger-plugin:       Unhandled error
@liskhq/lisk-framework-forger-plugin:       at process.uncaught (../../node_modules/jest-jasmine2/build/jasmine/Env.js:248:21)
@liskhq/lisk-framework-forger-plugin:     console.error
@liskhq/lisk-framework-forger-plugin:       Error: listen EADDRINUSE: address already in use 0.0.0.0:5000
@liskhq/lisk-framework-forger-plugin:           at Server.setupListenHandle [as _listen2] (net.js:1316:16)
@liskhq/lisk-framework-forger-plugin:           at listenInCluster (net.js:1364:12)
@liskhq/lisk-framework-forger-plugin:           at doListen (net.js:1501:7)
@liskhq/lisk-framework-forger-plugin:           at processTicksAndRejections (internal/process/task_queues.js:85:21)
@liskhq/lisk-framework-forger-plugin:       at process.uncaught (../../node_modules/jest-jasmine2/build/jasmine/Env.js:249:21)
@liskhq/lisk-framework-forger-plugin: FAIL test/functional/forger_info/forger_info_sync.spec.ts (6.103 s)
@liskhq/lisk-framework-forger-plugin:   ● Console
@liskhq/lisk-framework-forger-plugin:     console.error
@liskhq/lisk-framework-forger-plugin:       Unhandled error
@liskhq/lisk-framework-forger-plugin:       at process.uncaught (../../node_modules/jest-jasmine2/build/jasmine/Env.js:248:21)
@liskhq/lisk-framework-forger-plugin:     console.error
@liskhq/lisk-framework-forger-plugin:       Error: listen EADDRINUSE: address already in use 0.0.0.0:5000
@liskhq/lisk-framework-forger-plugin:           at Server.setupListenHandle [as _listen2] (net.js:1316:16)
@liskhq/lisk-framework-forger-plugin:           at listenInCluster (net.js:1364:12)
@liskhq/lisk-framework-forger-plugin:           at doListen (net.js:1501:7)
@liskhq/lisk-framework-forger-plugin:           at processTicksAndRejections (internal/process/task_queues.js:85:21)
@liskhq/lisk-framework-forger-plugin:       at process.uncaught (../../node_modules/jest-jasmine2/build/jasmine/Env.js:249:21)
@liskhq/lisk-framework-forger-plugin:   ● Forger Info Sync › should sync information from scratch on startup
@liskhq/lisk-framework-forger-plugin:     listen EADDRINUSE: address already in use 0.0.0.0:5000
@liskhq/lisk-framework-forger-plugin: PASS test/functional/voters.spec.ts (7.084 s)
@liskhq/lisk-framework-forger-plugin:   ● Console
@liskhq/lisk-framework-forger-plugin:     console.error
@liskhq/lisk-framework-forger-plugin:       Unhandled error
@liskhq/lisk-framework-forger-plugin:       at process.uncaught (../../node_modules/jest-jasmine2/build/jasmine/Env.js:248:21)
@liskhq/lisk-framework-forger-plugin:     console.error
@liskhq/lisk-framework-forger-plugin:       Error: listen EADDRINUSE: address already in use 0.0.0.0:5000
@liskhq/lisk-framework-forger-plugin:           at Server.setupListenHandle [as _listen2] (net.js:1316:16)
@liskhq/lisk-framework-forger-plugin:           at listenInCluster (net.js:1364:12)
@liskhq/lisk-framework-forger-plugin:           at doListen (net.js:1501:7)
@liskhq/lisk-framework-forger-plugin:           at processTicksAndRejections (internal/process/task_queues.js:85:21)
@liskhq/lisk-framework-forger-plugin:       at process.uncaught (../../node_modules/jest-jasmine2/build/jasmine/Env.js:249:21)
@liskhq/lisk-framework-http-api-plugin: PASS test/functional/blocks.spec.ts (26.28 s)
@liskhq/lisk-framework-http-api-plugin: PASS test/functional/node.spec.ts (6.899 s)
@liskhq/lisk-framework-http-api-plugin:   ● Console
@liskhq/lisk-framework-http-api-plugin:     console.error
@liskhq/lisk-framework-http-api-plugin:       Unhandled error
@liskhq/lisk-framework-http-api-plugin:       at process.uncaught (../../node_modules/jest-jasmine2/build/jasmine/Env.js:248:21)
@liskhq/lisk-framework-http-api-plugin:     console.error
@liskhq/lisk-framework-http-api-plugin:       Error: listen EADDRINUSE: address already in use 0.0.0.0:5000
@liskhq/lisk-framework-http-api-plugin:           at Server.setupListenHandle [as _listen2] (net.js:1316:16)
@liskhq/lisk-framework-http-api-plugin:           at listenInCluster (net.js:1364:12)
@liskhq/lisk-framework-http-api-plugin:           at doListen (net.js:1501:7)
@liskhq/lisk-framework-http-api-plugin:           at processTicksAndRejections (internal/process/task_queues.js:85:21)
@liskhq/lisk-framework-http-api-plugin:       at process.uncaught (../../node_modules/jest-jasmine2/build/jasmine/Env.js:249:21)
```

### Steps to reproduce

Run tests suit on Jenkins

### Which version(s) does this affect? (Environment, OS, etc...)

development

### PR DESCRIPTION
**Pull request by shuse2**
### What was the problem?

This PR resolves #6308 and resolves #6310

### How was it solved?

- Update Jenkinsfile to split the heavy test to run in different machine
  in sequence
- Remove outdated code
- Update plugin functional to run one by one

### How was it tested?

- Run jenkins multiple times

### CODE REVIEW

File: Jenkinsfile.sdk [hunk 1]
```diff
@@ -153,6 +143,15 @@ pipeline {
 						}
 					}
 				}
+			stage('Lisk Elements / @liskhq/lisk-p2p') {
```
Comment (fchavant) [inline Jenkinsfile.sdk:136]: Indentation looks wrong for this whole `stage` block.

File: Jenkinsfile.sdk [hunk 2]
```diff
@@ -235,35 +221,21 @@ pipeline {
...
-						setup()
-						nvm(getNodejsVersion()) {
-							sh 'npx lerna run test:integration --scope=@liskhq/lisk-framework-*'
+							agent { node { label 'lisk-framework' }}
+							steps {
+								setup()
+								nvm(getNodejsVersion()) {
+									sh 'npx lerna run test:integration --scope=@liskhq/lisk-framework-* --concurrency 1'
+								}
+							}
```
Comment (fchavant) [inline Jenkinsfile.sdk:231]: Indentation looks wrong.

File: Jenkinsfile.sdk [hunk 3]
```diff
@@ -3,26 +3,6 @@
 def setup(start_application=false) {
```
Comment (fchavant) [inline Jenkinsfile.sdk:3]:
```suggestion
def setup() {
```

File: framework/src/controller/controller.ts [hunk 4]
```diff
@@ -126,7 +126,7 @@ export class Controller {
 				? plugins
 				: [...Object.keys(this._inMemoryPlugins), ...Object.keys(this._childProcesses)];
 
-		const errors = [];
+		let hasError = false;
```
Comment (ManuGowda) [inline framework/src/controller/controller.ts:129]: 🤔 why is this here? it was merged in separate PR?

File: framework/src/testing/block_processing_env.ts [hunk 5]
```diff
@@ -256,6 +256,7 @@ export const getBlockProcessingEnv = async (
 		genesisBlock.header.id,
 		appConfig.genesisConfig.communityIdentifier,
 	);
+	removeDB(params.options?.databasePath);
```
Comment (ManuGowda) [inline framework/src/testing/block_processing_env.ts:259]: 🤔 the cleanup is doing removeDB, not sure why it doesn't work

**Review by ManuGowda — APPROVED** [review]

File: framework/src/testing/block_processing_env.ts (hunk 5 above)
Comment (shuse2) [reply framework/src/testing/block_processing_env.ts:259]: in particular instance, it was not removed properly and never able to rerun the test anymore, so i fixed it by ensuring the DB don't exist

File: framework/src/controller/controller.ts (hunk 4 above)
Comment (shuse2) [reply framework/src/controller/controller.ts:129]: without this PR, the test doesn't pass and without this fix, the deepscan won't pass, so i merged it here

**Review by fchavant — APPROVED** [review]
